src/main/target/common_pre.h

```diff
@@ -320,6 +320,7 @@
 #define USE_CRSF_CMS_TELEMETRY
 #define USE_CRSF_LINK_STATISTICS
 #define USE_RX_RSSI_DBM
+#define USE_RX_SNR_DBM
```

Comment:
This seems to have no use without CRSF, so it should be undefined in `common_post.h` if `USE_CRSF` isn't defined.

Reply:
Already in that file.

Reply:
Sorry, didn't spot.